deploy/005-remote-timelock.ts

```diff
@@ -1,7 +1,9 @@
+import { ethers } from "hardhat";
 import { DeployFunction } from "hardhat-deploy/types";
 import { HardhatRuntimeEnvironment } from "hardhat/types";
 
 import { SUPPORTED_NETWORKS } from "../helpers/deploy/constants";
+import { delayConfig } from "../helpers/deploy/deploymentConfig";
```

Comment:
I am personally partial to keeping configs local to the deployment scripts. This would mean that the code is duplicated for remote and source timelocks but I think that is acceptable given the context of the two deployments is different.

the remote timelocks use a different contract for hardhat. The source timelocks should probably use the same logic.